File: restyle.py
import os
import re

files_to_process = [
    r'd:\2\lyke-india\src\components\ui\sign-in-card-2.tsx',
    r'd:\2\lyke-india\src\components\ui\sign-up-card.tsx',
    r'd:\2\lyke-india\src\app\loading.tsx'
]

replacements = {
    r'bg-black': 'bg-[#FAF8F5]',
    r'from-purple-500/40': 'from-[#D2B48C]/40',
    r'via-purple-700/50': 'via-[#8B5A2B]/20',
    r'to-black': 'to-[#FAF8F5]',
    r'bg-purple-400/20': 'bg-[#D2B48C]/30',
    r'bg-purple-300/20': 'bg-[#C19A6B]/30',
    r'bg-white/5': 'bg-[#8B5A2B]/5',
    r'rgba\(255,255,255,0\.03\)': 'rgba(139,90,43,0.05)',
    r'rgba\(255,255,255,0\.05\)': 'rgba(139,90,43,0.08)',
    r'via-white': 'via-[#8B5A2B]',
    r'from-white/3': 'from-[#8B5A2B]/10',
    r'via-white/7': 'via-[#8B5A2B]/20',
    r'bg-black/40': 'bg-white/40',
    r'border-white/\[0\.05\]': 'border-[#8B5A2B]/10',
    r'linear-gradient\(135deg, white 0\.5px': 'linear-gradient(135deg, rgba(139,90,43,0.2) 0.5px',
    r'linear-gradient\(45deg, white 0\.5px': 'linear-gradient(45deg, rgba(139,90,43,0.2) 0.5px',
    r'border-white/10': 'border-[#8B5A2B]/20',
    r'bg-black/50': 'bg-white/60',
    r'from-white to-white/70': 'from-[#4A3728] to-[#8B5A2B]',
    r'from-white/10 to-transparent': 'from-[#8B5A2B]/10 to-transparent',
    r'from-white to-white/80': 'from-[#4A3728] to-[#5C4033]',
    r'text-white/60': 'text-[#5C4033]/70',
    r'bg-red-500/20': 'bg-red-100',
    r'border-red-500/30': 'border-red-200',
    r'text-red-200': 'text-red-600',
    r'text-white/40': 'text-[#8B5A2B]/60',
    r'text-white/30': 'text-[#8B5A2B]/50',
    r'text-white/70': 'text-[#8B5A2B]/90',
    r'text-white/80': 'text-[#8B5A2B]',
    r'text-white': 'text-[#4A3728]',
    r'border-white/20': 'border-[#8B5A2B]/30',
    r'focus:border-white/20': 'focus:border-[#8B5A2B]/40',
    r'focus:bg-white/10': 'focus:bg-white/80',
    r'border-t border-white/5': 'border-t border-[#8B5A2B]/10',
    r'bg-white text-black': 'bg-[#4A3728] text-[#FAF8F5]',
    r'bg-white/10': 'bg-[#4A3728]/10',
    r'from-white/0 via-white/30': 'from-[#FAF8F5]/0 via-[#FAF8F5]/30',
    r'border-black/70': 'border-[#FAF8F5]/70',
    r'hover:text-white': 'hover:text-[#4A3728]',
    r'checked:bg-white': 'checked:bg-[#4A3728]',
    r'checked:border-white': 'checked:border-[#4A3728]',
    r'focus:ring-white/30': 'focus:ring-[#4A3728]/30',
    r'text-black pointer-events-none': 'text-[#FAF8F5] pointer-events-none',
    r'border-t-purple-400': 'border-t-[#8B5A2B]',
    r'bg-white': 'bg-[#8B5A2B]',  # line underline in link
    r'bg-\[\#8B5A2B\]/5 border-transparent focus:border-\[\#8B5A2B\]/40': 'bg-transparent border-transparent focus:border-[#8B5A2B]/40',
    r'bg-\[\#4A3728\]/10 text-\[\#FAF8F5\] font-medium h-10 rounded-lg transition-all duration-300 flex items-center justify-center': 'bg-[#4A3728] text-[#FAF8F5] font-medium h-10 rounded-lg transition-all duration-300 flex items-center justify-center',
}

for file_path in files_to_process:
    if not os.path.exists(file_path):
        continue
    
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()
    
    for pattern, replacement in replacements.items():
        content = re.sub(pattern, replacement, content)
        
    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(content)

File: restyle.js
const fs = require('fs');
const path = require('path');

const filesToProcess = [
    path.join(__dirname, 'src', 'components', 'ui', 'sign-in-card-2.tsx'),
    path.join(__dirname, 'src', 'components', 'ui', 'sign-up-card.tsx'),
    path.join(__dirname, 'src', 'app', 'loading.tsx')
];

const replacements = [
    { pattern: /bg-black/g, replacement: 'bg-[#FAF8F5]' },
    { pattern: /from-purple-500\/40/g, replacement: 'from-[#D2B48C]/40' },
    { pattern: /via-purple-700\/50/g, replacement: 'via-[#8B5A2B]/20' },
    { pattern: /to-black/g, replacement: 'to-[#FAF8F5]' },
    { pattern: /bg-purple-400\/20/g, replacement: 'bg-[#D2B48C]/30' },
    { pattern: /bg-purple-300\/20/g, replacement: 'bg-[#C19A6B]/30' },
    { pattern: /bg-white\/5/g, replacement: 'bg-[#8B5A2B]/5' },
    { pattern: /rgba\(255,255,255,0\.03\)/g, replacement: 'rgba(139,90,43,0.05)' },
    { pattern: /rgba\(255,255,255,0\.05\)/g, replacement: 'rgba(139,90,43,0.08)' },
    { pattern: /via-white/g, replacement: 'via-[#8B5A2B]' },
    { pattern: /from-white\/3/g, replacement: 'from-[#8B5A2B]/10' },
    { pattern: /via-white\/7/g, replacement: 'via-[#8B5A2B]/20' },
    { pattern: /bg-black\/40/g, replacement: 'bg-white/40' },
    { pattern: /border-white\/\[0\.05\]/g, replacement: 'border-[#8B5A2B]/10' },
    { pattern: /linear-gradient\(135deg, white 0\.5px/g, replacement: 'linear-gradient(135deg, rgba(139,90,43,0.2) 0.5px' },
    { pattern: /linear-gradient\(45deg, white 0\.5px/g, replacement: 'linear-gradient(45deg, rgba(139,90,43,0.2) 0.5px' },
    { pattern: /border-white\/10/g, replacement: 'border-[#8B5A2B]/20' },
    { pattern: /bg-black\/50/g, replacement: 'bg-white/60' },
    { pattern: /from-white to-white\/70/g, replacement: 'from-[#4A3728] to-[#8B5A2B]' },
    { pattern: /from-white\/10 to-transparent/g, replacement: 'from-[#8B5A2B]/10 to-transparent' },
    { pattern: /from-white to-white\/80/g, replacement: 'from-[#4A3728] to-[#5C4033]' },
    { pattern: /text-white\/60/g, replacement: 'text-[#5C4033]/70' },
    { pattern: /bg-red-500\/20/g, replacement: 'bg-red-100' },
    { pattern: /border-red-500\/30/g, replacement: 'border-red-200' },
    { pattern: /text-red-200/g, replacement: 'text-red-600' },
    { pattern: /text-white\/40/g, replacement: 'text-[#8B5A2B]/60' },
    { pattern: /text-white\/30/g, replacement: 'text-[#8B5A2B]/50' },
    { pattern: /text-white\/70/g, replacement: 'text-[#8B5A2B]/90' },
    { pattern: /text-white\/80/g, replacement: 'text-[#8B5A2B]' },
    { pattern: /text-white/g, replacement: 'text-[#4A3728]' },
    { pattern: /border-white\/20/g, replacement: 'border-[#8B5A2B]/30' },
    { pattern: /focus:border-white\/20/g, replacement: 'focus:border-[#8B5A2B]/40' },
    { pattern: /focus:bg-white\/10/g, replacement: 'focus:bg-white/80' },
    { pattern: /border-t border-white\/5/g, replacement: 'border-t border-[#8B5A2B]/10' },
    { pattern: /bg-white text-black/g, replacement: 'bg-[#4A3728] text-[#FAF8F5]' },
    { pattern: /bg-white\/10/g, replacement: 'bg-[#4A3728]/10' },
    { pattern: /from-white\/0 via-white\/30/g, replacement: 'from-[#FAF8F5]/0 via-[#FAF8F5]/30' },
    { pattern: /border-black\/70/g, replacement: 'border-[#FAF8F5]/70' },
    { pattern: /hover:text-white/g, replacement: 'hover:text-[#4A3728]' },
    { pattern: /checked:bg-white/g, replacement: 'checked:bg-[#4A3728]' },
    { pattern: /checked:border-white/g, replacement: 'checked:border-[#4A3728]' },
    { pattern: /focus:ring-white\/30/g, replacement: 'focus:ring-[#4A3728]/30' },
    { pattern: /text-black pointer-events-none/g, replacement: 'text-[#FAF8F5] pointer-events-none' },
    { pattern: /border-t-purple-400/g, replacement: 'border-t-[#8B5A2B]' },
    { pattern: /bg-white/g, replacement: 'bg-[#8B5A2B]' }, // This handles the link underline correctly due to earlier text-white replacements 
    { pattern: /bg-\[\#8B5A2B\]\/5 border-transparent focus:border-\[\#8B5A2B\]\/40/g, replacement: 'bg-transparent border-transparent focus:border-[#8B5A2B]/40' },
    { pattern: /bg-\[\#4A3728\]\/10 text-\[\#FAF8F5\] font-medium h-10 rounded-lg transition-all duration-300 flex items-center justify-center/g, replacement: 'bg-[#4A3728] text-[#FAF8F5] font-medium h-10 rounded-lg transition-all duration-300 flex items-center justify-center' }
];

filesToProcess.forEach(filePath => {
    if (fs.existsSync(filePath)) {
        let content = fs.readFileSync(filePath, 'utf-8');
        replacements.forEach(({ pattern, replacement }) => {
            content = content.replace(pattern, replacement);
        });
        fs.writeFileSync(filePath, content, 'utf-8');
        console.log(`Processed ${filePath}`);
    } else {
        console.log(`Skipped ${filePath}`);
    }
});
